main.go

```diff
@@ -204,7 +213,7 @@ func runSuite(suite TestSuite) []TestResult {
 		}
 
 		unused := vars.Unused()
-		if len(unused) != 0 {
+		if !dryRunFlag && len(unused) != 0 {
```

Comment:
Because variables are set based on HTTP response body and if we do not make an actual HTTP call - we won't have this response, so we will have all vars unused.